src/interpreter/vm.h

```diff
@@ -146,20 +161,16 @@ namespace verona::interpreter
     void check_type(const Value& value, std::vector<Value::Tag> expected);
 
     const Code& code_;
-    rt::Alloc* alloc_;
-    bool verbose_;
-
-    /**
-     * Instruction Pointer
-     */
-    size_t ip_;
+    rt::Alloc* const alloc_;
+    const bool verbose_;
 
     /**
      * Address of the currently executing instruction.
```

Comment:
`start_ip_` is "Address of the currently executing function"?

Reply:
No, the comment is correct. `start_ip` gets updated in each iteration of `dispatch_loop`.
On the other hand there, it wasn't being saved correctly by `execute_finaliser`, so I've fixed that.